wallets/serverless.yml

```diff
@@ -196,3 +196,17 @@ functions:
         - ${file(./config.${self:provider.stage}.json):VPC2}
     events:
       - schedule: rate(2 minutes)
+
+  open-channel-by-third-party:
+    warmup: true
```

Comment:
can we remove warmup as we are not using plugin, so no need to add it right now. We can make issue and track this for other lambdas as well.

Reply:
We don't need it here. But we are using in other places. So, I have added the warm-up plugin dependency.